internal/utils/ci_providers.go

```diff
@@ -277,18 +285,33 @@ func extractGithubActions() map[string]string {
 		branch = branchOrTag
 	}
 
-	url := fmt.Sprintf("https://github.com/%s/commit/%s/checks", os.Getenv("GITHUB_REPOSITORY"), os.Getenv("GITHUB_SHA"))
+	serverUrl := os.Getenv("GITHUB_SERVER_URL")
+	if serverUrl == "" {
+		serverUrl = "https://github.com"
+	}
+
+	rawRepository := fmt.Sprintf("%s/%s", serverUrl, os.Getenv("GITHUB_REPOSITORY"))
+	pipelineId := os.Getenv("GITHUB_RUN_ID")
+	commitSha := os.Getenv("GITHUB_SHA")
+
 	tags[constants.CIProviderName] = "github"
-	tags[constants.GitRepositoryURL] = fmt.Sprintf("https://github.com/%s.git", os.Getenv("GITHUB_REPOSITORY"))
-	tags[constants.GitCommitSHA] = os.Getenv("GITHUB_SHA")
+	tags[constants.GitRepositoryURL] = fmt.Sprintf("%s.git", rawRepository)
+	tags[constants.GitCommitSHA] = commitSha
 	tags[constants.GitBranch] = branch
 	tags[constants.GitTag] = tag
 	tags[constants.CIWorkspacePath] = os.Getenv("GITHUB_WORKSPACE")
-	tags[constants.CIPipelineID] = os.Getenv("GITHUB_RUN_ID")
+	tags[constants.CIPipelineID] = pipelineId
 	tags[constants.CIPipelineNumber] = os.Getenv("GITHUB_RUN_NUMBER")
 	tags[constants.CIPipelineName] = os.Getenv("GITHUB_WORKFLOW")
-	tags[constants.CIPipelineURL] = url
-	tags[constants.CIJobURL] = url
+	tags[constants.CIJobURL] = fmt.Sprintf("%s/commit/%s/checks", rawRepository, commitSha)
+
+	attempts := os.Getenv("GITHUB_RUN_ATTEMPT")
+	if attempts == "" {
+		tags[constants.CIPipelineURL] = fmt.Sprintf("%s/actions/runs/%s", rawRepository, pipelineId)
+	} else {
+		tags[constants.CIPipelineURL] = fmt.Sprintf("%s/actions/runs/%s/attempts/%s", rawRepository, pipelineId, attempts)
+	}
```

Comment:
same concern about the leading slash for these

Reply:
fixed in 334a193